.github/workflows/build.yml

```diff
@@ -84,6 +84,9 @@ jobs:
           nohup mvn quarkus:dev > app.log 2>&1 &
           PID=$(echo $!)
 
+          # Wait until the app is up and running
+          tail -f app.log | sed '/Installed features/ q'
```

Comment:
the tail command will also print the content of the Maven dev command and will exit once the "Installed features" pharse is there.